inc/namespace.php

```diff
@@ -2,12 +2,13 @@
 
 namespace Altis\Security;
 
+use Altis\Security\PHP_Basic_Auth;
 use const Altis\ROOT_DIR;
 use function Altis\get_config;
-use function Altis\get_environment_type;
 
 function bootstrap() {
-	add_action( 'plugins_loaded', __NAMESPACE__ . '\\on_plugins_loaded', 1 );
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\on_plugins_loaded' );
```

Comment:
Can we reinstate the priority here? I'm concerned that this was likely set for a reason so if there isn't a reason to change it we should leave it as is.

Reply:
D'oh, I thought that was something that I added because I was messing with priorities.